src/taskgraph/run-task/run-task

```diff
@@ -1033,6 +1033,24 @@ def _display_python_version():
         b"setup", b"Python version: %s\n" % platform.python_version().encode("utf-8")
     )
 
+def _display_subprocess_python_version():
+
+    python_versions = ["python","python2","python3"]
+
+    for python_executable in python_versions:
+        try:
+            version = subprocess.check_output(
+                [python_executable, "-c", "import platform; print(platform.python_version())"],
+                text=True
+            ).strip()
+
+            print_line(
+                b"setup", b"Subprocess %s version: %s\n" % (python_executable.encode(), version.encode("utf-8"))
```

Comment:
nit: If you remove the `text=True` then `version` will be a byte string already and you won't need to re-encode it.